packages/ripple-binary-codec/src/enums/index.ts

```diff
@@ -70,22 +72,34 @@ export class Bytes {
 class BytesLookup {
   constructor(types: Record<string, number>, readonly ordinalWidth: number) {
     Object.entries(types).forEach(([k, v]) => {
-      this[k] = new Bytes(k, v, ordinalWidth)
-      this[v.toString()] = this[k]
+      this.add(k, v)
     })
   }
 
+  /**
+   * Add a new name value pair to the BytesLookup. Does not work properly if either name or value already
+   * exist within the BytesLookup.
+   *
+   * @param name - A human readable name for the field.
+   * @param value - The numeric value for the field.
+   */
+  add(name: string, value: number): void {
+    this[name] = new Bytes(name, value, this.ordinalWidth)
```

Comment:
`this.ordinalWidth` doesn't ever appear to be initialized.

Reply:
It comes from the constructor and is a required parameter.